docs/quickstart/cpp.md

```diff
@@ -19,6 +19,11 @@ working example.</p>
 To install gRPC on your system, follow the [instructions to build from
```

Comment:
instead of adding more instructions in this doc, we should point to https://github.com/grpc/grpc/tree/master/src/cpp#make instead, which has the info about how to install with different build systems (and links to the building.md doc too).

Reply:
Then you're suggesting to insert the link you showed in place of the one already present? Or as an additional pointer?

Reply:
I basically wanted to replace the existing link. I've done so in this PR and I'm going to merge.